apps/firelens-stability/collections/ecs-firelens-stability-tests/golden-path/cases/no-onepod-high-throughput.json

```diff
@@ -2,7 +2,7 @@
     "definitions": {
         "include_onepod": false,
         "kbps_throughput_tcp": 5000,
-        "kbps_throughput_tail": 5000,
+        "kbps_throughput_tail": 10,
```

Comment:
I know this is because it has high memory usage... but its not really a high throughput test anymore with this :/